src/pylero/document.py

```diff
@@ -316,6 +318,38 @@ def query(
             lst_doc = [Document(suds_object=doc) for doc in docs]
             return lst_doc
 
+    @classmethod
+    def export_pdf(cls, uri, exportPdfProperties, output_dir, output_name: "output"):
```

Comment:
Better set output_name as empty "", then check if it's empty string use the uri to get the doc name and set it as output_name.

Reply:
for the output_dir could set default value as "/tmp"

Reply:
done